### ISSUE TITLE
Upgrade URL for download Plex updates

### ISSUE THREAD

**Issue by jheredianet** [linuxserver/docker-plex#26]
Hey, I saw that plex download has changed its url. It was https://downloads.plex.tv/ and now its https://plex.tv/downloads
When you restart the container, it is not starting since its in a loop redirecting to the URL.
Please upgrade it.

**Comment by zaggash** [comment]
I think the problem is not here.
Since the URL only change for the browser location.

The Package URL is still the same.
eg: https://downloads.plex.tv/plex-media-server/0.9.15.6.1714-7be11e1/Plex-Media-Server-0.9.1506.1714-7be11e1-en-US.exe

**Comment by lonix** [comment]
Can Confirm that package name is the same. do you have a log of your issue of sorts, so we can identify the problem ?

**Comment by jheredianet** [comment]
Hi, I wanted to get the logs, but it's working now. It's downloading the plex update, installing and running. So it seems that the error I got yesterday is fixed now. Thanks.

----Partial final Log -----
2016-03-17 13:17:29 (24.2 MB/s) - ‘/tmp/plexmediaserver_0.9.16.0.1754-23623fb_amd64.deb’ saved [128181166/128181166]

Reading package lists...
Building dependency tree...
Reading state information...
The following packages will be REMOVED:
  plexmediaserver*
0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
After this operation, 306 MB disk space will be freed.
(Reading database ... 17785 files and directories currently installed.)
Removing plexmediaserver (0.9.15.2.1663-7efd046) ...
Purging configuration files for plexmediaserver (0.9.15.2.1663-7efd046) ...
dpkg: warning: while removing plexmediaserver, directory '/etc/systemd/system' not empty so not removed
Processing triggers for ureadahead (0.100.0-16) ...
Processing triggers for mime-support (3.54ubuntu1.1) ...
Processing triggers for libc-bin (2.19-0ubuntu6.6) ...
Reading package lists... Done
Building dependency tree
Reading state information... Done
Building data structures... Done
  Selecting previously unselected package plexmediaserver.
(Reading database ... 14185 files and directories currently installed.)
Preparing to unpack .../plexmediaserver_0.9.16.0.1754-23623fb_amd64.deb ...
Unpacking plexmediaserver (0.9.16.0.1754-23623fb) ...
Setting up plexmediaserver (0.9.16.0.1754-23623fb) ...
OK
Processing triggers for ureadahead (0.100.0-16) ...
Processing triggers for mime-support (3.54ubuntu1.1) ...
Building data structures... Done
‘/defaults/plexmediaserver’ -> ‘/etc/default/plexmediaserver’
*** Running /etc/my_init.d/90_chown_plex_owned_files.sh...
*** Running /etc/my_init.d/999_advanced_script.sh...
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 123
Starting Avahi daemon
Starting Plex Media Server.
Starting dbus-daemon
Found user 'avahi' (UID 104) and group 'avahi' (GID 107).
Successfully dropped root privileges.
avahi-daemon 0.6.31 starting up.
dbus_bus_get_private(): Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
WARNING: Failed to contact D-Bus daemon.
avahi-daemon 0.6.31 exiting.
6 3000 /config/Library/Application Support
8192
Mar 17 13:17:43 elitemc syslog-ng[136]: syslog-ng starting up; version='3.5.3'
Starting Avahi daemon
Found user 'avahi' (UID 104) and group 'avahi' (GID 107).
Successfully dropped root privileges.
avahi-daemon 0.6.31 starting up.
No service file found in /etc/avahi/services.
Joining mDNS multicast group on interface veth218eff8.IPv6 with address fe80::fc83:ff:fe5b:33d1.
New relevant interface veth218eff8.IPv6 for mDNS.
Joining mDNS multicast group on interface docker0.IPv6 with address fe80::42:9dff:fe0d:1a48.
New relevant interface docker0.IPv6 for mDNS.
Joining mDNS multicast group on interface docker0.IPv4 with address 172.17.0.1.
New relevant interface docker0.IPv4 for mDNS.
Joining mDNS multicast group on interface enp0s25.IPv6 with address fe80::2e41:38ff:fe17:33f1.
New relevant interface enp0s25.IPv6 for mDNS.
Joining mDNS multicast group on interface enp0s25.IPv4 with address 10.10.10.2.
New relevant interface enp0s25.IPv4 for mDNS.
Network interface enumeration completed.
Registering new address record for fe80::fc83:ff:fe5b:33d1 on veth218eff8.*.
Registering new address record for fe80::42:9dff:fe0d:1a48 on docker0.*.
Registering new address record for 172.17.0.1 on docker0.IPv4.
Registering new address record for fe80::2e41:38ff:fe17:33f1 on enp0s25.*.
Registering new address record for 10.10.10.2 on enp0s25.IPv4.
Registering HINFO record with values 'X86_64'/'LINUX'.
Server startup complete. Host name is ubuntu.local. Local service cookie is 3075147950.
********  Started WebTools V2.1 on Linux  **********
